### PR TITLE
✨Support <amp-delight-player dock>

### PR DESCRIPTION
**Pull request by alanorozco**

### CODE REVIEW

File: extensions/amp-delight-player/0.1/amp-delight-player.js [hunk 1]
```diff
@@ -192,11 +197,13 @@ class AmpDelightPlayer extends AMP.BaseElement {
 
   /** @override */
   createPlaceholderCallback() {
-    const placeholder = this.element.ownerDocument.createElement('div');
+    const html = htmlFor(this.element);
+    const placeholder =
+        html`<div placeholder><amp-img layout=fill></amp-img></div>`;
```
Comment (alanorozco) [inline extensions/amp-delight-player/0.1/amp-delight-player.js]: @aghassemi Even though this makes it technically get the correct placeholder URL, the `src` pattern below appears to be broken :(

File: extensions/amp-video-docking/0.1/amp-video-docking.js [hunk 2]
```diff
@@ -1708,7 +1708,6 @@ export class VideoDocking {
       this.getControls_().reset();
 
       [
-        element,
```
Comment (alanorozco) [inline extensions/amp-video-docking/0.1/amp-video-docking.js]: Caught this while manually testing. Same as #20912.

File: extensions/amp-video-docking/0.1/test/validator-amp-video-docking-amp-delight-player.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<!--
+  Copyright 2018 The AMP HTML Authors. All Rights Reserved.
```
Comment (honeybadgerdontcare) [inline extensions/amp-video-docking/0.1/test/validator-amp-video-docking-amp-delight-player.html]: s/2018/2019

File: extensions/amp-delight-player/0.1/amp-delight-player.js (hunk 1 above)
Comment (aghassemi) [reply extensions/amp-delight-player/0.1/amp-delight-player.js]: @xymw has the Url for poster changed?

**Pull request merged**